tests/FARFF/corrector.sh

```diff
@@ -14,6 +14,6 @@ PPPATH="../../"
 export PYTHONPATH=$PYTHONPATH:$PPPATH
 #python -m pyProbeParticle.FARFF
 #python3 -m pyProbeParticle.Corrector
-python3 -m pyProbeParticle.CorrectionLoop --job train
+python3 -m pyProbeParticle.ml.CorrectionLoop --job train
```

Comment:
Did you tried to run correction loop ?

Reply:
I tried running the script at examples/CorrectionLoopGraphene/test_CorrectionLoop.py and at least it completes successfully.